templates/example_flowsom_clustering.ipynb

```diff
@@ -18,6 +18,7 @@
    "outputs": [],
```

Comment:
<p>This didn't really address my previous comment about separating data that needs to be generated each time, and data that only needs to be generated once. </p><p> </p><p>In addition, I think it makes more sense to put the SOM-specific directories after the cell where we specify which channels are going to be included in this run. We should give people the option to name the folder an informative name, and otherwise default to the timestamp-based scheme you have here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>